catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -126,7 +136,30 @@ private CqlQueryResponse executeCqlQuery(CqlRequest cqlRequest)
                 filterBuilder);
 
         Stopwatch stopwatch = Stopwatch.createStarted();
-        QueryResponse response = catalogFramework.query(request);
+        QueryResponse response = null;
+
+        if (configuration.isDisableLocalCatalog()) {
+            Set<String> sourceIds;
+            if (CollectionUtils.isNotEmpty(request.getSourceIds())) {
+                sourceIds = new HashSet<>(request.getSourceIds());
+            } else {
+                sourceIds = new HashSet<>(catalogFramework.getSourceIds());
+            }
+            sourceIds.remove(catalogFramework.getId());
+            request = new QueryRequestImpl(request.getQuery(),
+                    false,
```

Comment:
❓ It looks like for a cache query we want to set `isEnterprise` to true, will this conflict with that?
https://github.com/codice/ddf/blob/9939ef34e50441223d9114ef4c320a97f840106c/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java#L132-L135